### ISSUE TITLE
Disable extension

### ISSUE THREAD

**Issue by sh7dm** [eclipse-theia/theia#7558]
### Description
Is there any way to disable extension?

### Reproduction Steps
I couldn't find any way to disable the extension without uninstalling it.

**OS and Theia version:**
OS: Ubuntu 19.10
Theia: `master`

**Diagnostics:**

**Comment by vince-fugnitto** [comment]
@sh7dm an extension can either be removed through:
- removing it from the `package.json` (won't work for extensions that depend on the extension you want to remove and are part of the application)
  - ex: remove B, but A which is included in your app requires B
- removing the extension's functionality by removing it through dependency injection in your custom extension

**Comment by sh7dm** [comment]
No, my extension isn't built-in. Probably that's a plugin (VS Code extension).

**Comment by vince-fugnitto** [comment]
> No, my extension isn't built-in. Probably that's a plugin (VS Code extension). It's stored in `~/.theia/extensions/` directory.

You're referring to VS Code plugins which you've added to the application?

**Comment by sh7dm** [comment]
@vince-fugnitto I mean ones installed from Open-VSX, they aren't from the Theia app itself, they were installed using Open-VSX:
![image](https://user-images.githubusercontent.com/19504461/79121145-476ec280-7d9d-11ea-9d17-b30e0b673337.png)

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto I mean ones installed from Open-VSX, they aren't from the Theia app itself, they were installed using Open-VSX:
> ![image](https://user-images.githubusercontent.com/19504461/79121145-476ec280-7d9d-11ea-9d17-b30e0b673337.png)

At the moment, the only option is to uninstall the extension, you may open an issue in order to track the feature of **disabling** them instead.

**Comment by sh7dm** [comment]
I did so :). This is that issue. Could you please add related labels etc. ?

**Comment by vince-fugnitto** [comment]
> I did so :). This is that issue. Could you please add related labels etc. ?

I've updated the issue so it's clearer for others when they reference it.

**Comment by akosyakov** [comment]
Duplicate of https://github.com/eclipse-theia/theia/issues/6528

VSX registry is just UI, we should implement proper enablement management in the plugin extension first.

**Comment by sgraband** [comment]
`disablePlugin`/`enablePlugin` fully implemented with UI commands and context menus.
